cmd/proxy-server/main.go

```diff
@@ -395,18 +389,10 @@ func run(log *logrus.Entry) error {
 	}
 	defer storageConn.Close()
 
-	proxyConn, err := grpc.Dial(proxyAddr,
-		grpc.WithTimeout(10*time.Second),
-		grpc.WithInsecure())
-	if err != nil {
-		return err
-	}
-	defer proxyConn.Close()
-
 	router := &web.Router{
 		RolesHandler:      web.Adapt(rolesHandler(log, cfg.OpenPolicyAgent.Host), web.OtelMW(tp, "roles")),
 		TokenHandler:      web.Adapt(refreshTokenHandler(pb.NewTenantServiceClient(tenantConn), log), web.OtelMW(tp, "refresh")),
-		AdminTokenHandler: web.Adapt(refreshAdminTokenHandler(pb.NewAuthServiceClient(proxyConn), log), web.OtelMW(tp, "refresh")),
+		AdminTokenHandler: web.Adapt(refreshAdminTokenHandler(log), web.OtelMW(tp, "refresh")),
```

Comment:
The OtelMW will create spans called "refresh" for both tenant and admin refresh. Can we make these "tenant_refresh" and "admin_refresh"?

Reply:
fixed